plugins/com.google.cloud.tools.eclipse.appengine.standard.java8.test/META-INF/MANIFEST.MF

```diff
@@ -4,7 +4,7 @@ Bundle-Name: Google Cloud Tools for App Engine Standard Java 8 Tests
 Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.standard.java8.test
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.standard.java8
 Bundle-Version: 0.1.0.qualifier
-Bundle-Vendor: Google, Inc.
+Bundle-Vendor: Google Inc.
```

Comment:
If we do care about this, we should probably add a test for it to the BasePluginXmlTest.

Reply:
Added a test. Let's see if all the tests will succeed.